src/lib/utils.ts

```diff
@@ -2,41 +2,68 @@ import lzString from 'lz-string';
 const { decompressFromEncodedURIComponent, compressToEncodedURIComponent } = lzString;
 
 const parseQuery = (
-	input: string
+    input: string
 ): {
-	error: boolean;
-	query: string;
-	variables: string;
-	operationName: string;
-	queryString: string;
+    error: boolean;
+    message: string; 
```

Comment:
It doesn't look like `message` is being used anywhere. Should it be added to the UI?

Reply:
Probably, I'll fix it at home.